feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
OK! In that case, I'm fine with deferring a fix for this to a separate PR. Let's just make sure an issue is filed for it.